stdlib/3/hashlib.pyi

```diff
@@ -36,5 +37,8 @@ def new(name: str, data: _DataType = ...) -> Hash: ...
 algorithms_guaranteed = ...  # type: AbstractSet[str]
 algorithms_available = ...  # type: AbstractSet[str]
 
-# New in version 3.4
-def pbkdf2_hmac(name: str, password: _DataType, salt: _DataType, rounds: int, dklen: int = ...) -> bytes: ...
+# New in version 3.4, rounds renamed to iterations in 3.5
+if sys.version_info >= (3, 5):
+    def pbkdf2_hmac(hash_name: str, password: _DataType, salt: _DataType, iterations: int, dklen: Optional[int] = ...) -> bytes: ...
+else:
+    def pbkdf2_hmac(name: str, password: _DataType, salt: _DataType, rounds: int, dklen: Optional[int] = ...) -> bytes: ...
```

Comment:
As far as I can tell 3.4 also uses the name "iteratons" (https://github.com/python/cpython/blob/3.4/Lib/hashlib.py#L148 and https://github.com/python/cpython/blob/3.4/Modules/_hashopenssl.c#L598). So we should just use the version of the function in the >= 3.5 branch, but add a guard `if sys.version_info >= (3, 4)`.